<launch>
    <arg name="config_path" default="$(find vins_estimator)/../config/tum_rgbd/tum_fr3_mpi.yaml" />
    <arg name="vins_path" default="$(find vins_estimator)/../config/../" />
    <arg name="rgb" default="/my_robot_0/camera_link/0/rgb/image_raw" />
    <arg name="depth" default="/my_robot_0/camera_link/0/depth/image_raw" />

    <!-- remap from="/camera/color/image_raw" to="/d400/color/image_raw" /-->
    <!-- remap from="/camera/color/image_raw" to="/my_robot_0/camera_link/0/rgb/image_raw" /-->
    <remap from="/rgb" to="$(arg rgb)" />
    <remap from="/depth" to="$(arg depth)" />

    <remap from="/camera/color/image_raw" to="/my_robot_0/camera_link/0/rgb/image_raw" />
    <include file="$(find yolo_ros)/launch/yolo_service.launch">
    </include>

    <arg name="manager_name" default="nodelet_manager_pc" />
    <node pkg="nodelet" type="nodelet" name="$(arg manager_name)" args="manager" output="screen"/>

    <node pkg="nodelet" type="nodelet" name="EstimatorNodelet" args="load vins_estimator/EstimatorNodelet $(arg manager_name)" output="screen">
        <param name="config_file" type="string" value="$(arg config_path)" />
        <param name="vins_folder" type="string" value="$(arg vins_path)" />
    </node>

    <node pkg="tf" type="static_transform_publisher" name="world_to_map" args="0.0 0.0 0.0 0.0 0.0 0.0 /world /map 100" />
<!-- 
    <node pkg="nodelet" type="nodelet" name="PoseGraphNodelet" args="load pose_graph/PoseGraphNodelet $(arg manager_name)" output="screen">
        <param name="config_file" type="string" value="$(arg config_path)"/>
        <param name="visualization_shift_x" type="int" value="0"/>
        <param name="visualization_shift_y" type="int" value="0"/>
        <param name="skip_cnt" type="int" value="0"/>
        <param name="skip_dis" type="double" value="0"/>
    </node> -->

</launch>